evictionfree/housing_court.py

```diff
@@ -36,6 +36,14 @@ def get_housing_court_info_for_user(user: JustfixUser) -> Optional[HousingCourtI
             name=BOROUGH_COURT_NAMES[oi.borough], email=BOROUGH_EMAILS[oi.borough]
         )
 
-    # TODO: Need to figure out what email to send this to!
+    # NY state really wants the county name in the subject line, so we only want
+    # to send them an email if we've geocoded the user's address and know what
+    # county they're in.
+    if oi.lookup_county():
```

Comment:
This also gives us time to properly geocode some of the users who have run into wacky edge cases in our geocoder, such as the one mentioned in #1989.

Reply:
ohhh ok gotcha— so @toolness this means that we may have some upstate court emails "queued" bc the geocoding didn't work properly? In which case we may need to intervene and manually put in the county?

Reply:
Correct, or rather we won't be manually putting in the county, but instead running that `verify_addresses` command from #1990 to either geocode them properly or, uh, delete the users or something (in the cases where someone from TN snuck into EFNY lol), and then at that point the email would have enough info to get sent.

The one wrinkle here is that, right now at least, we need to manually write and run a script that goes through all users who _have_ sent a declaration but _haven't_ had an email sent to housing court yet (e.g. all our outside-NYS users right now) and see if we can send their letters.  I might just modify the `process_declarations` command added in #1946, which runs on a regular basis, to do this for us.

Reply:
Update! I've just added such functionality to `process_declarations` in #1994.